Sources/Features/Shared/CloudKit/CloudKitInterface.swift

```diff
@@ -354,21 +552,51 @@ extension CKDatabaseOperation: CKDatabaseOperationType {
     public typealias Database = CKDatabase
 }
 
+@available(iOS 10.0, OSX 10.12, tvOS 10.0, watchOS 3.0, *)
+extension CKAcceptSharesOperation: CKAcceptSharesOperationType, AssociatedErrorType {
+
+    // The associated error type
+    public typealias Error = CloudKitError
+}
+
 /// Extension to have CKDiscoverAllContactsOperation conform to CKDiscoverAllContactsOperationType
 #if !os(tvOS)
+@available(iOS, introduced=8.0, deprecated=10.0, message="Use CKDiscoverAllUserIdentitiesOperation instead")
+@available(OSX, introduced=10.10, deprecated=10.12, message="Use CKDiscoverAllUserIdentitiesOperation instead")
+@available(watchOS, introduced=2.0, deprecated=3.0, message="Use CKDiscoverAllUserIdentitiesOperation instead")
```

Comment:
Interesting - didn't spot that they'd deprecated these operations. This is great :)